UnitySDK/Assets/ML-Agents/Examples/3DBall/TFModels/3DBallHardLearning.nn.meta

```diff
@@ -1,7 +1,10 @@
 fileFormatVersion: 2
 guid: 8be33caeca04d43498913448b5364f2b
 ScriptedImporter:
+  internalIDToNameTable: []
```

Comment:
Should these `.meta` files be committed?

Reply:
yes

Reply:
They may have changes based on the editor version I was using, so I will double check that and revert them if that's the case.